test/test_data/static/hosts/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+main2:
+  host1:
+    foo2: bar2
```

Comment:
the file name must always match the hostname... so this file should be called host1.yml